<launch>

  <arg name="namespace" default="pandora_vision/hole_detector" />

    <node
      name="synchronizer_node"
      type="synchronizer_node"
      pkg="pandora_vision_hole_detector"
      respawn="False"
      output="screen"
      ns="$(arg namespace)"
    />

    <node
      name="depth_node"
      type="depth_node"
      pkg="pandora_vision_hole_detector"
      respawn="False"
      output="screen"
      ns="$(arg namespace)"
    />

    <node
      name="rgb_node"
      type="rgb_node"
      pkg="pandora_vision_hole_detector"
      respawn="False"
      output="screen"
      ns="$(arg namespace)"
    />

    <node
      name="hole_fusion_node"
      type="hole_fusion_node"
      pkg="pandora_vision_hole_detector"
      respawn="False"
      output="screen"
      ns="$(arg namespace)"
    />

</launch>
